packages/i18n/src/useI18n.ts

```diff
@@ -0,0 +1,22 @@
+import {useI18nController} from './Controller';
+
+const useI18n = (namespaces: string[]) => {
+  const context = useI18nController();
+
+  const t = (key: string, options: object = {}) => {
```

Comment:
Shouldn't we type this `options` better? It uses `context.t` second argument type.